WcaOnRails/app/jobs/dump_developer_database.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class DumpDeveloperDatabase < ApplicationJob
+  queue_as :default
+
+  def perform
+    # Create developer database dump every 3 days.
+    last_developer_db_dump = Timestamp.find_or_create_by(name: 'last_developer_db_dump')
+    if last_developer_db_dump.date.nil? || last_developer_db_dump.date < 3.days.ago
```

Comment:
Ah, the `nil` case... How very mildly annoying...

Simple local fix: Change to `Timestamp.find_or_create_by(name: 'last_developer_db_dump', date: 4.days.ago)`.

Bigger better fix: Make the `timestamps.date` column default to `0` rather than `NULL`.

That should mean (I haven't tested) Jan 1 1970, which the the typical usage simply work.